### PR TITLE
Missed french text

### LINKED ISSUE #318: Missed French in Alpha 6 on CERB + EI Regular and CERB + EI Sickness
The blue callouts were missed in translation of changes two benefits cards:
- CERB + EI Regular 
- CERB + EI SIckness.

French is now in Figma under Alpha 6.

### PR DESCRIPTION
**Pull request by CalvinRodo**
Closes #318  Adds Missing French translations
Also attempts to fix an issue with the staging branch not being seen as a release branch by the tagging action. 

EI Sickness and CERB old Result

image

New version

image

EI Regular and CERB old Result 

image

New Version 

image

### CODE REVIEW

File: .github/workflows/build-deploy-staging.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
         id: tag
         uses: anothrNick/github-tag-action@1.13.0
         env:
-          RELEASE_BRANCH: 'staging'
+          RELEASE_BRANCH: staging
```
Comment (dsamojlenko) [inline .github/workflows/build-deploy-staging.yml:19]: seems unrelated - intentional?

Comment (CalvinRodo) [reply .github/workflows/build-deploy-staging.yml:19]: it was intentional, it is unrelated, but I wanted to get this fix in and figure it was small enough to not need to bother with a separate PR.

Comment (dsamojlenko) [reply .github/workflows/build-deploy-staging.yml:19]: 👍
